From a5dd8f7ddc00f0f4e58f56b729b7e0066edf4e71 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 12 Aug 2020 10:15:52 +0100
Subject: [PATCH] overlays: Regenerate upstream-pi4

The recent modification to vc4-kms-v3d-pi4 also results in a change
to the Pi 4 version of the upstream overlay.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/upstream-pi4-overlay.dts | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm/boot/dts/overlays/upstream-pi4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-pi4-overlay.dts
@@ -128,6 +128,12 @@
 		};
 	};
 	fragment@20 {
+		target = <&dvp>;
+		__overlay__ {
+			status = "okay";
+		};
+	};
+	fragment@21 {
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
